sky/adaptors/azure.py

```diff
@@ -14,19 +17,40 @@
 
 _session_creation_lock = threading.RLock()
 
+# There is no programmatic way to get current account details anymore.
+# https://github.com/Azure/azure-sdk-for-python/issues/21561
+
+
+def _get_account():
+    result = run('az account show -o json',
+                 stdout=subprocess.PIPE,
+                 stderr=subprocess.PIPE)
+
+    if result.returncode != 0:
+        error_message = result.stderr.decode()
+        print(f'Error executing command: {error_message}')
+        raise RuntimeError(
+            'Failed to execute az account show -o json command.')
```

Comment:
nit: we usually use the following context manager to disable traceback for simplicity of the output.

https://github.com/skypilot-org/skypilot/blob/5a0ecc703c80ada615274993ade116ba7ed2570c/sky/utils/ux_utils.py#L26-L40